src/store/reducers/room.ts

```diff
@@ -91,6 +99,16 @@ function userReducer(state: ConnectedRoomMap = initialState, action: RoomAction)
                 },
             };
 
+        case RECEIVE_MESSAGE:
```

Comment:
I really have to learn from you how to use Redux properly (when refactoring `MainHome.tsx` more later). This look clean and easy to understand